rohit/widgets/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+//import Accordion from './componets/Accordion';
```

Comment:
Rohit, please pay attention when fixing errors. Fix not just one error or one line, but look around for more of such errors.